plugins/CoreHome/lang/en.json

```diff
@@ -11,6 +11,7 @@
         "CloseWidgetDirections": "You can close this widget by clicking on the 'X' icon at the top of the widget.",
         "ChooseX": "Choose %1$s",
         "DataForThisReportHasBeenPurged": "The data for this report is more than %s months old and has been purged.",
+        "DataForThisReportHasBeenDisabled": "Segmentation is currently disabled for this report. Please check %1$sthis FAQ%2$s for more details.",
```

Comment:
@tsteur Not sure if this will be a feature we are going to use on cloud maybe. In that case I was wondering if linking an FAQ in the message would actually be helpful, as the user can't change that 🤔